app/view_models/issue_detail.rb

```diff
@@ -75,6 +75,49 @@ def essays
     EssayQuery.published_essays_for_issue(issue).collect { | e | EssayLink.new(e) }
   end
 
+  def has_essays_of_style(essay_style_id)
+    EssayQuery.published_essays_for_issue_and_essay_style(issue, EssayStyle.find_by(slug: essay_style_id).id).count > 0
```

Comment:
```suggestion
    style = EssayStyle.find_by(slug: essay_style_id)
    if style.present?
      EssayQuery.published_essays_for_issue_and_essay_style(issue, style.id).count > 0
    else
      false
    end
```
Should allow the method to return false if the requested essay style doesn't exist, and the unit test will pass.